src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
We could execute the config file, but this won't be able if php-cs-fixer is ran in PHAR modus. In that case, the `PhpCsFixer\Config` won't be available. 
I do understand why it crashes, but isn't it more logical if it runs in intersect modus without a finder, it runs on the specified files only? However, this would require a change in php-cs-fixer2.